addon/pom.xml

```diff
@@ -5,7 +5,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.vaadin.stefan</groupId>
     <artifactId>fullcalendar2</artifactId>
-    <version>6.2.1</version>
+    <version>6.2.2</version>
```

Comment:
Please add -SNAPSHOT to the version as it is not the final build. Thanks :)

Reply:
I updated the master regarding the versions.

Reply:
Added -SNAPSHOT to versions